app/main.py

```diff
@@ -1 +1,9 @@
-# write your code here
+def copy_file(command: str) -> None:
+    copy_file_command, sour_file, des_file = command.split()
```

Comment:
What if the command is something like cp file.txt (only 2 parts)?
Add some check for elements amount.